contracts/currency/currency.abi

```diff
@@ -14,11 +14,22 @@
         {"name":"memo", "type":"string"}
       ]
     },{
+     "name": "create",
+     "base": "",
+     "fields": [
+        {"name":"issuer", "type":"account_name"},
+        {"name":"maximum_supply", "type":"asset"},
+        {"name":"can_freeze", "type":"uint8"}
```

Comment:
are `can_freeze|recall|whitelist` in use anywhere?  I don't see any test cases or uses of it (other than trivial initialization).  Those seem like implementation details or at least part of a larger bag of properties a currency may/may-not have.  As this example contract serves as the only basis for a currency standard I think we should consider not taking a position on these qualities until they are more fleshed out.

Reply:
I'm not implementing this as an example, I am implementing this as part of a broader currency contract that implements the features of BitShares.  

It is incomplete at this stage.

Reply:
Cool Stuff.... More light in a deep rabbit hole.....

Reply:
If it is incomplete and/or not universal(or exemplary) can we move the implementation (`eosiolib\currency.hpp`)from into the contract specific folder and/or name it something more specific like "bitshares currency" ?  

Without any documentation, I would expect this to be the current normative iteration of the currency standard based on the structure of the repo.